include/deal.II/grid/grid_tools.h

```diff
@@ -1501,7 +1501,13 @@ namespace GridTools
     const std::vector<bool> &                              marked_vertices = {},
     const RTree<std::pair<Point<spacedim>, unsigned int>> &used_vertices_rtree =
       RTree<std::pair<Point<spacedim>, unsigned int>>{},
-    const double tolerance = 1.e-10);
+    const double tolerance = 1.e-10,
+    const RTree<
+      std::pair<BoundingBox<spacedim>,
+                typename Triangulation<dim, spacedim>::active_cell_iterator>>
+      &rtree = RTree<std::pair<
```

Comment:
Can you give this a more expressive name? See e.g. the `used_vertices_rtree` above.